app/tests/spec/views/mixins/modal-settings-panel-mixin.js

```diff
@@ -0,0 +1,97 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+define([
+  'chai',
+  'sinon',
+  'underscore',
+  'views/mixins/modal-settings-panel-mixin',
+  'views/base',
+  'lib/metrics',
+  'stache!templates/test_template',
+  '../../../lib/helpers'
+], function (Chai, sinon, _, ModalSettingsPanelMixin, BaseView,
+    Metrics, TestTemplate, TestHelpers) {
+  'use strict';
+
+  var assert = Chai.assert;
+
+  var ModalSettingsPanelView = BaseView.extend({
+    template: TestTemplate
+  });
+
+  _.extend(ModalSettingsPanelView.prototype, ModalSettingsPanelMixin);
```

Comment:
Since the mixin is mixed in using Cocktail, Cocktail should probably be used here just to ensure there are not subtle, unexpected differences (like with the pseudo-function merging).